subsys/suit/storage/src/suit_storage_nrf54h20.c

```diff
@@ -88,11 +94,14 @@ struct suit_storage_nordic {
 			/* Application MPI Backup and digest. */
 			struct suit_storage_mpi_app app_mpi_bak;
 
+			/* Flags used for selecting the boot path. */
+			struct suit_storage_flags flags;
+
 			/* The last SUIT update report. */
-			uint8_t report_0[160];
+			uint8_t report_0[144];
 
 			/* Reserved for Future Use. */
-			uint8_t reserved[160];
+			uint8_t reserved[144];
```

Comment:
This is an error - only one of the `report_0`/`reserved`  fields should be reduced, as `flags` is a 16bytes field.
Probably `reserved` should be reduced to 144 and `report_0` left at 160 bytes.

The way it is now the secure domain fails to boot.

Reply:
By the way, at first it seems unclear where does the number 144 precisely come from... Perhaps it would be more readable to add the number of occupied bytes next to each field? It could also help to avoid such errors
Like:
`struct suit_storage_flags flags; /* 16 bytes */

Reply:
Changed to 160.